textattack/search_methods/particle_swarm_optimization.py

```diff
@@ -40,6 +40,7 @@ def __init__(
         self.pop_size = pop_size
         self.post_turn_check = post_turn_check
         self.max_turn_retries = 20
+        self.is_black_box = True
```

Comment:
Do you need this and the @property below? I think you can leave this and remove the @property methods, if I recall correctly.

Reply:
Yeah I was debating whether to set it as regular attribute or @property, and decided to go with @property and forgot to delete it. I think we should just keep it as @property since that's how I do it in the parent `SearchMethod` class.